### PR TITLE
setup redux

### PR DESCRIPTION
**Pull request by b4yc**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import { Redirect, Route } from 'react-router-dom';
 import { IonApp, IonIcon, IonLabel, IonRouterOutlet, IonTabBar, IonTabButton, IonTabs } from '@ionic/react';
 import { IonReactRouter } from '@ionic/react-router';
 import { ellipse, square, triangle } from 'ionicons/icons';
-import Tab1 from './pages/Tab1';
-import Tab2 from './pages/Tab2';
-import Tab3 from './pages/Tab3';
+import Navigation from './pages/Navigation';
+import Car from './pages/Car';
+import Music from './pages/Music';
```
Comment (RatikKapoor) [inline src/App.tsx:8]: nice!

**Review by RatikKapoor — CHANGES_REQUESTED** [review]
- Use yarn instead of npm, delete package-lock.json since we'll be sticking with yarn
- Might also be helpful to delete node_modules, then run `yarn install` after deleting package-lock.json

File: src/serviceWorker.ts [hunk 2]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```
Comment (RatikKapoor) [inline src/serviceWorker.ts]: Might want to keep this so ESLint ignores the contents of this file. It's not usually modified from what I know

**Review by RatikKapoor — APPROVED** [review]
:shipit:

**Review by dhyxy — APPROVED** [review]
🤠

**Pull request merged**